src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Models/TargetDependencyViewModel.cs

```diff
@@ -26,7 +26,9 @@ static ProjectTreeFlags GetCachedFlags(TargetFramework targetFramework)
                 return ImmutableInterlocked.GetOrAdd(
                     ref s_configurationFlags,
                     targetFramework.FullName,
-                    fullName => DependencyTreeFlags.TargetNode.Add($"$TFM:{fullName}"));
+                    fullName => DependencyTreeFlags.TargetNode
+                        .Add($"$TFM:{fullName}")
+                        .Add(ProjectTreeFlags.Common.VirtualFolder));
```

Comment:
>VirtualFolder [](start = 53, length = 13)

What does adding this flag change in the behavior of the node?

Reply:
This PR actually _removes_ `VirtualFolder` from most nodes (via `DependencyTreeFlags.DependencyFlags`). This addition here keeps it present on a node for which it makes sense.

The flag itself isn't used very widely in CPS. It's used to short circuit some tree scanning. The docs on the flag itself aren't very helpful.